src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -706,6 +709,10 @@ else if (multipleViolationsBelowMatcher.matches()) {
                         inputConfigBuilder.addViolation(actualLineNumber, null);
                     });
         }
+        else if (violationsDefault.matches()) {
+                    final int violationLineNum = lineNo + 1;
+                    inputConfigBuilder.addViolation(violationLineNum, null);
```

Comment:
Please move code to left, to have no extra spaces on left.

Reply:
https://ci.appveyor.com/project/Checkstyle/checkstyle/builds/50256080/job/drp3uw8ceedrvos0#L111

Please always run `mvn verify` before pushing to GitHub, and resolve all violations from such execution

Reply:
Done. Do I fix all of the parts where CI complains that violations differ?

Reply:
Yes, we need green CI.
```
Violation lines for C:\projects\checkstyle\src\test\resources\com\puppycrawl\tools\checkstyle\checks\javadoc\writetag\InputWriteTagSeverity.java differ.
missing (1): 13
---
expected   : [13]
but was    : []

```

Looks like there's is more problems, please investigate